telemetry/definitions/commonDefinitions.json

```diff
@@ -541,6 +541,11 @@
             "type": "string",
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
+        },
+        {
+            "name": "codewhispererPerceivedLatencyDuration",
```

Comment:
Does it make sense to use the already defined `duration` attribute instead? We can put the description in the metric description.

Reply:
updated, thanks!